### DISCUSSION TITLE
Category by Colour

### DISCUSSION THREAD

**Discussion by 1devils1** [rgriebl/brickstore#921] [Ideas]
Hi All,

Is there a way or future plans to show all elements in a particular colour per category.

So I select plate modified as the category - Now I want to see everything available in Reddish Brown.

Cheers

**Comment by rgriebl** [comment]
That's implemented for a long time already. Click "Reddish Brown", then press the "Color Lock" button in the top-right corner.

**Reply by 1devils1** [reply]
THANKS!! :)
